pip_outdated/__init__.py

```diff
@@ -29,6 +34,13 @@ def main():
     from .check_outdated import check_outdated
     from .print_outdated import print_outdated
     requires = find_require(args.file)
-    outdated_results = check_outdated(requires)
-    print_outdated(outdated_results, args.quiet)
+    
+    import aiohttp
```

Comment:
any particular reason why you don't want to import this at the top?

Reply:
It's a habit to improve the initialization time of the CLI. Large modules are loaded on demand.